### PR TITLE
chore(main/lsd): Enable shell completions

### PR DESCRIPTION
**Pull request by Biswa96**

### CODE REVIEW

**Comment by Biswa96** [comment]
@licy183 Could you provide the reason why the zlib workaround was added in c1563167830b077953244fe9727d7fe5d084021a commit?

**Comment by Grimler91** [comment]
> @licy183 Could you provide the reason why the zlib workaround was added in [c156316](https://github.com/termux/termux-packages/commit/c1563167830b077953244fe9727d7fe5d084021a) commit? It is not required now.

I guess same reason we have that workaround for rust:
https://github.com/termux/termux-packages/blob/865657dd45207aabc388f5358dd62ed4af41568d/packages/rust/build.sh#L61-L66

Rust mixes host libraries and cross-compiled libraries and compilation fails. 
If you can't reproduce issue then maybe workaround can be removed (maybe [Truboxl's rust cleanup](https://github.com/termux/termux-packages/commit/efc0a1cf291d6aa93ae4f07f42645df303ef932e) helped improve things)

File: packages/lsd/build.sh [hunk 1]
```diff
@@ -1,35 +1,21 @@
...
 termux_step_pre_configure() {
-	termux_setup_rust
-
-	mv $TERMUX_PREFIX/lib/libz.so.1{,.tmp}
-	mv $TERMUX_PREFIX/lib/libz.so{,.tmp}
-
-	local _CARGO_TARGET_LIBDIR="target/${CARGO_TARGET_NAME}/release/deps"
-	mkdir -p "${_CARGO_TARGET_LIBDIR}"
-
-	ln -sfT "$(readlink -f "${TERMUX_PREFIX}"/lib/libz.so.1.tmp)" \
```
Comment (TomJo2000) [inline packages/lsd/build.sh:21]: This I'd like to have tested on device before merging.

Comment (Biswa96) [reply packages/lsd/build.sh:21]: May the device be with you and powered on :upside_down_face:

**Pull request merged**